<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="status">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="status_statu_lable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>控制状态：</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_speed_lable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>控制速度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_status">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_speed">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="Status_speed_lable2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>实际转速：</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_true_speed">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>90</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="status_motor_turn_R">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>正</string>
    </property>
   </widget>
   <widget class="QPushButton" name="status_motor_turn_L">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>110</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>反</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="control">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>131</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="control_button_on">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/on.png</normaloff>:/images/on.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="control_botton_off">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/off.png</normaloff>:/images/off.png</iconset>
    </property>
   </widget>
   <widget class="QSlider" name="control_speed_slider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0  1  2  3  4  5(00)               </string>
    </property>
   </widget>
   <widget class="QLabel" name="control_motor_image">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/images/motor.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
